unittests/bad_mock_server.py

```diff
@@ -0,0 +1,124 @@
+from flask import Flask, request, Response
+import json
+import os
+import re
+from constants import GOOD_SERVER_URL
+
+good_server_host = GOOD_SERVER_URL.split("://")[1].split(":")[0]
+good_server_port = GOOD_SERVER_URL.split("://")[1].split(":")[1].replace("/","")
+
+app = Flask(__name__)
+
+'''
+HTTP codes
+200:"Success"
+206:"Success. Filtered Subsequence"
+303:"Redirect to a url where sequence can be retrieved"
+400:"Bad Request",
+401:"Unauthorized",
+404:"Not Found",
+406:"Not Acceptable",
+416:"Range Not Satisfiable",
+501:"Not Implemented"
+'''
+
+@app.route('/sequence/service-info', methods=['GET'])
```

Comment:
the endpoint name should follow the[ DRS specification](https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.2.0/docs/#tag/GA4GH-Service-Registry). Instead of "/sequence/service-info", it should be "/ga4gh/drs/v1/"+"service-info"

Reply:
Changed to fit in with the DRS specification.